src/test/policyestimator_tests.cpp

```diff
@@ -130,14 +158,35 @@ BOOST_AUTO_TEST_CASE(BlockPolicyEstimates)
         for (int j = 0; j < 10; j++) { // For each fee multiple
             for (int k = 0; k < 4; k++) { // add 4 fee txs
                 tx.vin[0].prevout.n = 10000*blocknum+100*j+k;
+                {
+                    LOCK2(cs_main, mpool.cs);
+                    mpool.addUnchecked(entry.Fee(feeV[j]).Time(Now<NodeSeconds>()).Height(blocknum).FromTx(tx));
+                    // Since TransactionAddedToMempool callbacks are generated in ATMP,
+                    // not addUnchecked, we cheat and create one manually here
+                    const int64_t virtual_size = GetVirtualTransactionSize(*MakeTransactionRef(tx));
+                    const NewMempoolTransactionInfo tx_info = NewMempoolTransactionInfo(MakeTransactionRef(tx),
```

Comment:
ac82f58e45421aa5aec1d3e6e992d1c369472a10
a bit less verbose
```suggestion
                    const NewMempoolTransactionInfo tx_info{MakeTransactionRef(tx),
```

Reply:
Done